tests/e2e/specs/EventBasedGateway.spec.js

```diff
@@ -59,7 +59,7 @@ describe('Event-Based Gateway', () => {
     getElementAtPosition(taskPosition)
       .click()
       .then($task => {
-        getCrownButtonForElement($task, 'delete-button').click();
+        getCrownButtonForElement($task, 'delete-button').click({ force: true });
```

Comment:
This was required since the copy crown button pushed the delete crown button off screen.

Reply:
Oof that scroll-to thing